docs/source/advanced_usage.rst

```diff
@@ -108,7 +108,7 @@ If you want the validator to be executed only for certain types of repositories,
 
     from fiasko_bro import tokenized_validator
 
-    @tokenized_validator(token='min_max_challenge')
+    @tokenized_validator(list_of_tokens=['min_max_challenge'])
```

Comment:
There are multiple interpretations to the name: will a validator matching _all_ of the markers run or _any_? We should rename it to `run_if_any`. That said, we should make corresponding `run_if_all` decorator and `run_if` for a single token case. Make sure to factor out the common parts.